main.js

```diff
@@ -0,0 +1,306 @@
+const IPA_key = "1e877084";
+const wrapMovies = document.querySelector(".wrap-movies");
+const paginationWrap = document.querySelector(".pagination-wrap");
+const headerTitle = document.querySelector(".header-title");
+
+const mySearch = async () => {
+  try {
+    let current;
+    let movies = [];
+    const input = document.querySelector("#search").value;
+    const moviesResponse = await axios.get(
+      `http://www.omdbapi.com/?apikey=${IPA_key}&s=${input}&page=${current}`
+    );
+    movies = await moviesResponse.data.Search;
+    pageCleener();
+    renderMovies(movies);
+    pagination();
+  } catch (error) {
+    invalidResponseHandler();
+  }
+};
+
+function renderMovies(data) {
+  data.forEach((dataMovie) => {
+    const { Title, Year, Type, Poster, imdbID } = dataMovie;

```

Comment:
Nice, as improvement you can define your own names for variables, to remove capital letters